packages/@lwc/engine-server/src/index.ts

```diff
@@ -8,7 +8,8 @@
 import './polyfills';
 
 // Renderer initialization -------------------------------------------------------------------------
-import './initializeRenderer';
+import * as renderer from './initializeRenderer';
```

Comment:
not sure if we want to just export an object, or to deal with exotic namespace objects via `* as` syntax.